<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌번호</string>
    </property>
   </widget>
   <widget class="QComboBox" name="accComboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="gubunComboBox">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>160</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="tradeGubunComboBox">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>205</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="itemCodeTextEdit">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>70</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>35</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>종 목 명</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sellPushButton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>294</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>매 도</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyPushButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>294</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>매  수</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="orderNumberTextEdit">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>250</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>80</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>종목코드</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchItemButton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>25</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>조    회</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="priceSpinBox">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>160</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="volumeSpinBox">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>116</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>126</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>수     량</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelPushButton">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>294</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>취 소</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>170</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>가     격</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changePushButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>294</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>정 정</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>215</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>거래구분</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="searchItemTextEdit">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>25</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>260</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>원주문번호</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>791</width>
      <height>231</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>계좌잔고</string>
     </attribute>
     <widget class="QTableWidget" name="stocklistTableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>761</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>미체결주문</string>
     </attribute>
     <widget class="QTableWidget" name="outstandingTableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>761</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>549</y>
      <width>791</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="totalBuyingPriceLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>추정예탁자산</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>총평가금액</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="lineWidth">
        <number>4</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>총매입금액</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>총평가손익금액</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="balanceAssetLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="currentTotalPriceLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="totalEstimateProfitLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>540</y>
      <width>471</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="chartLayout"/>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>490</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>봉 갯수</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="candlenumberTextEdit">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>490</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="charPushButton">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>490</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>조회</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>801</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>조건식/자동매매설정</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>조건식 작동시작</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>매수/매도 구분</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="startTimeEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="endTimeEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>~</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="conditionTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>241</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="autoTradeGubunComboBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addAutoTradePushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>추  가</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeAutoTradePushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>제  거</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="autoTradeconditionTableWidget">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>341</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="conditionSearchPushButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>50</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>조건검색시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="autoTradePushButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>180</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>자동매매시작</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>691</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>조건검색종목</string>
    </property>
    <widget class="QTableWidget" name="conditionItemTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>671</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1269</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
